special-interest-groups/sig-tools/member-list.md

```diff
@@ -0,0 +1,23 @@
+# SIG Tools Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2020-03-02
+
+## Active Contributors
+
+Members and their contributions:
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+
+## Tech Leads
+
+Leads:
+
+* [GregoryIan](https://github.com/GregoryIan), [PingCAP](https://pingcap.com/en/)
```

Comment:
@kennytm is willing to take charge of the tools-sig, please update it

Reply:
done